### ISSUE TITLE
Water removed from claim when creating a claim

### ISSUE THREAD

**Issue by SimCraftAdmin** [GriefPrevention/GriefPrevention#516]
### What happened:
When areas over water were claimed, a chunk of water was taken out of the area of the claim. While abandoning a claim, chunks of the contents of that claim are removed.

### What you expected:
When I Claim water - No Chunk removal (Clean)
When I Abandon - No Chunk Corruption

### Steps to reproduce:
1. Claim Water
You will likely notice that a chunk of water is removed
1. Abandon a Claim
2. It is likely that Parts of the content in that claim are removed.

### Server and GriefPrevention version:
16.11.6

### Stack trace/error or server log
(Player) Moved Too Quickly! (When Chunks were Removed)
### GriefPrevention config.yml

```
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/
GriefPrevention:
  SeaLevelOverrides:
    world: -1
    world_nether: -1
    world_the_end: -1
    Beach: -1
    Kingdom: -1
    survival: -1
    Mountania: -1
    Sand_City: -1
  Claims:
    Mode:
      Sand_City: Disabled
      Mountania: Disabled
      Kingdom: Disabled
      world: Creative
      world_nether: Disabled
      world_the_end: Disabled
      Beach: Disabled
      survival: Disabled
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: true
    LockTrapDoors: true
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 100
    Claim Blocks Accrued Per Hour:
      Default: 100
    Max Accrued Claim Blocks:
      Default: 2000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    PortalGenerationRequiresPermission: false
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
  Spam:
    Enabled: true
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: true
      world_nether: true
      world_the_end: true
      Beach: true
      Kingdom: true
      survival: true
      Mountania: true
      Sand_City: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: true
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  LimitPistonsToLandClaims: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - DIRT
    - GRASS_BLOCK
    - GRASS
    - FERN
    - DEAD_BUSH
    - COBBLESTONE
    - GRAVEL
    - SAND
    - GLASS
    - GLASS_PANE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false

```

### Plugin list (if applicable):
1. Dynmap
2. LuckPerms
3. PlugMan
4. WorldEdit
5. EssentialsX
6. Multiverse-Core (Grief Prevention Does not Like Additional Worlds)
7. DiscordSRV

**Comment by RoboMWM** [comment]
Creative claims do stuff to prevent liquids from "leaking" out of the claim, as water/lava can destroy/create blocks.

**Comment by RoboMWM** [comment]
I don't think any cleanup task should be occurring on claim (maybe on a resize though). Please be more specific by what you mean when a "chunk" of water is removed.

**Comment by 112RG** [comment]
The cleanup task is broken. As I still haven't fixed the water restore. But I didn't know claims regen when you abandon them

**Comment by RoboMWM** [comment]
In creative they always do, in survival it's only if you enable automatic restoration. But it shouldn't be happening on claim... so will have to doublecheck that. (May happen on resize, but waiting on if it's on resize or just on create.)

**Comment by 112RG** [comment]
Ah yeah I can see the confusion in

> When areas over water were claimed, a chunk of water was taken out of the area of the claim. While abandoning a claim, chunks of the contents of that claim are removed

But the regen task does remove water ATM since the logic for 1.13 is gonna be all new

**Comment by zagarjr** [comment]
This is still a thing...  Had this happen in my daughters' new world, and I'm using the latest version of GP

**Comment by RoboMWM** [comment]
@zagarjr The guy has not clarified this issue since asking questions to clarify his report. If you wish to chime in on an existing issue either provide the bug report template information here, or create a new issue.

**Comment by Jikoo** [comment]
Since OP is an account created to make this issue with no other activity in the 5 years since, going to close this. If anyone is still experiencing this issue, please create a new issue with more accurate reproduction steps.
